webapp/src/db.ts

```diff
@@ -4,7 +4,9 @@ import * as Promise from "bluebird";
 
 const PouchDB = require("pouchdb");
 /* tslint:disable:no-submodule-imports TODO(tslint) */
-require('pouchdb/extras/memory');
+PouchDB.plugin(require('pouchdb-adapter-cordova-sqlite'));
```

Comment:
Do we need these?

Reply:
So far it builds and runs fine without those modules. I included it since it was part of the original pouchdb.